Cargo.lock

```diff
@@ -3558,18 +3558,18 @@ checksum = "77af24da69f9d9341038eba93a073b1fdaaa1b788221b00a69bce9e762cb32de"
 
 [[package]]
 name = "openssl-src"
-version = "111.25.0+1.1.1t"
+version = "300.3.1+3.3.1"
```

Comment:
Upgrading to OpenSSL 3.x may downgrade performance, see https://github.com/tikv/tikv/blob/0a7fb7cdb61bb8731c4eba9404cc877c336d0b5e/deny.toml#L60

> I upgrade openssl, because cargo build failed complaining about a vulnerability found in the previous version.

Please validate whether the vulnerability impacts TiKV. If it does not, we can ignore it, see https://github.com/tikv/tikv/pull/17292

Reply:
Thx, Done